### ISSUE TITLE
iPhone 6 / 6 plus sizes

### ISSUE THREAD

**Issue by chrysb** [rafaelconde/ios8-ui-kit#8]
It would be great to have these templates for the new, larger dimensions of iPhone.

**Comment by rafaelconde** [comment]
I completely agree, and I plan to do so right after I get an iPhone 6 for myself (so I can get the screenshots the way I need them). 

In the meantime I did the Homescreen for the 6 and 6 Plus, it's something :p

**Comment by Spacelapp** [comment]
+1

**Comment by edwellbrook** [comment]
+1
